frontend/build-configs/constants/index.ts

```diff
@@ -7,4 +7,8 @@ export const BAYERN_STAGING_ID = 'staging.bayern.ehrenamtskarte.app'
 export const NUERNBERG_PRODUCTION_ID = 'nuernberg.sozialpass.app'
 export const NUERNBERG_STAGING_ID = 'staging.nuernberg.sozialpass.app'
 export const KOBLENZ_PRODUCTION_ID = 'koblenz.sozialpass.app'
-export const KOBLENZ_STAGING_ID = 'staging.koblenz.sozialpass.app'
\ No newline at end of file
+export const KOBLENZ_STAGING_ID = 'staging.koblenz.sozialpass.app'
+
+export const QUERY_PARAM_NAME = 'Name'
```

Comment:
i would prefer english names for this, as i think this is quite standard to do so :thinking:

Reply:
We always used german names for columns in the project i think so i kept this